applications/FluidDynamicsApplication/python_scripts/apply_slip_process.py

```diff
@@ -40,6 +45,15 @@ def __init__(self, Model, settings ):
             node.SetSolutionStepValue(KratosMultiphysics.IS_STRUCTURE,0,1.0)
             node.SetValue(KratosMultiphysics.Y_WALL,0.0)
 
+        if self.navier_slip_active:
+            navier_slip_length = settings["uniform_navier_slip_length"].GetDouble()
+            for node in self.model_part.Nodes:
+                node.SetValue(KratosMultiphysics.FluidDynamicsApplication.SLIP_LENGTH, navier_slip_length)
+        else:
```

Comment:
Why do you need to save the value is the Navier-Slip condition is not active.

Reply:
The high value of the slip_length was intended as the mechanism to switch the Navier slip contribution "off". Exactly here, the question came up whether it would be better to use a flag (or sth. comparable) to avoid the calculation of the Navier slip contribution a-priori in cases where it is not needed.

Reply:
For the sake of performance I'd rather go for the flag.